lib/internal/fs/rimraf.js

```diff
@@ -259,10 +265,18 @@ function _rmdirSync(path, options, originalErr) {
               i < tries &&
               options.retryDelay > 0) {
             sleep(i * options.retryDelay);
+          } else {
+            // The file is already gone.
+            if (err.code === 'ENOENT') {
+              return;
+            }
+            throw err;
```

Comment:
nit: we could avoid nested `if` here.
```suggestion
      } else if (err.code !== 'ENOENT') {
        throw err;
      } else {
        // The file is already gone.
        return;
```